testsuite/features/core/proxy_branch_network.feature

```diff
@@ -21,6 +21,14 @@ Feature: Setup SUSE Manager for Retail branch network
     And I manually install the "bind" formula on the server
     And I synchronize all Salt dynamic modules on "proxy"
 
+@proxy
+@private_net
+  # WORKAROUND
```

Comment:
ok please put bsc# if you create a bug

Reply:
From what I know it shouldn't be necessary, the package version is already released for all other relevant systems (Leap 15.2, SLE 15 SP3). I haven't find reason for Leap 15.3, but the whole situation currently doesn't sound like bug to me.